frontend/src/App.vue

```diff
@@ -55,13 +55,13 @@ export default {
             overflow scroll
             border solid 1px $-left-border-color
             background $-left-border-color
-            min-height 300px
+            min-height 1300px
```

Comment:
Is there a reason to increase the min-height so much?

Reply:
changed it back. As I showed you, I will add scrolling functionality for the long graph.